customization-tests/swagger/README.md

```diff
@@ -33,6 +33,40 @@ license-header: MICROSOFT_MIT_SMALL
 add-context-parameter: true
 models-subpackage: implementation.models
 context-client-method-parameter: true
-customization-jar-path: target/bodycomplex-customization-1.0.0-beta.1.jar
-customization-class: fixtures.bodycomplex.customization.BodyComplexCustomization
+customization-class: BodyComplexCustomization
 ```
+
+### Customization
+```java
+public class BodyComplexCustomization extends Customization {
```

Comment:
Would the user be able to write this code in a README without the help of compilable code?

Reply:
Without an IDE it's definitely harder to write this. But that's a tradeoff users will have to make. They can opt for easier customization coding but have to set up a maven project, or opt for easier setup but lose IDE features.